ams/AssetMigrationTracker.cs

```diff
@@ -64,6 +65,12 @@ internal class AssetMigrationTracker : IMigrationTracker<BlobContainerClient, As
         internal const string MigrateResultKey = "MigrateResult";
         internal const string ManifestNameKey = "ManifestName";
         internal const string OutputPathKey = "OutputPath";
+        
+        public async Task<bool> DeleteMigrationStatus(BlobContainerClient container,string MigratedBlobName)
+        {
+            return await container.DeleteBlobIfExistsAsync(MigratedBlobName);       
+        }
+
```

Comment:
This can be removed, no code is actually calling this method.  The method in IMigrationTracker can also be removed.

Even if it is called, the implementation is wrong here, DeleteMigrationStatus should not delete the entire blob.